### ISSUE TITLE
Tile widgets Exceed container bounds when scrolling/ paginated navigation

### ISSUE THREAD

**Issue by Bar343** [FlutterFlow/flutterflow-issues#891]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When tile widgets such as the checkbox are placed inside a list view or repeating column, the tiles are visible outside the bound of the component when scrolling. T

https://github.com/FlutterFlow/flutterflow-issues/assets/52680054/7dd40b66-5365-40d9-b8ba-af0b45540568

his occurs whether or not containers are set to clip content.

### Expected Behavior

widgets should stay within bounds of the container like al the other widgets do.

### Steps to Reproduce

1. Create a new page.
2. Add a button on the new page.
3. Create a new component.
4. In the new component and a pageview inside a container.
5. Add a list view as the content for a page in the pageview and 
6. Add a checkbox widget to the list view.
7. Repeat the checkbox dynamically through a backend query (make sure the list is long enough to scroll)
8. place the list on eth second page of the pageview as well so that you can navigate between.
9. On the main page and add a 'show bottom sheet' action to the button
10. select the component as the bottom sheet to display.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4Ol/LfsM5IsfVA+KrqccAwpDkXQ0w2U5ZN18J+Vk00GOLNGYsIYPfBSFBVZ7zmenxiM2eImnsy087Zkvz+G/oCFxeBUblq3btUdAqXdzmWaanQEcyoOUZDEdNnCEib1MKnuAx9FPpuciBl2F2Qeuq9SgDYQ4O3DEQjD/H9LoXA+l6mLQzVETtVziRecCfn

### Context

It's a visual eye sore and makes the interface seem less polished. Also, as the selected values are being passed to an API a Tile widget is preferred as the values can be dynamically selected as opposed to using a container.

### Visual documentation

https://xfzm-ouwd-peos.n2.xano.io/vault/sIFfslJo/G-sCdcuBPLhnFNhJUZmtOcdhzG0/bNZP5A../FlutterFlow+-+Tile+error.mp4

### Environment

```markdown
- FlutterFlow version: 3.1
- Platform: Windows Desktop
- Browser name and version: Edge  113.0.1774.42
- Operating system and version affected: Windows 10
```

**Comment by wenkaifan0720** [comment]
Hi @Bar343. Kudos to the attention of detail! I did some testing locally and I think the `ListTile` family just does not play well with the `PageView`. The good thing is, we have individual `Checkbox` and you can make a component with it to recreate the UI you have right now.

**Comment by Bar343** [comment]
Yes this is a workaround. But much mor difficult to link values since the
checkbox itself only has true or false values.

I can of ourse create a read only input field to link the dynamic  value to
wich i can then select within a logic flow.

Hopefully this does get fixed un flutter flows end.

Thank you.

**Comment by wenkaifan0720** [comment]
I think this is rather a Flutter issue, as I tested this with a local Flutter project and tried to peel off as many widgets around the `ListTile` as possible.
